pom.xml

```diff
@@ -175,7 +175,33 @@
                     </mapping>
                 </configuration>
             </plugin>
-       </plugins>
+
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-dependency-plugin</artifactId>
+                <version>2.8</version>
+                <executions>
+                    <execution>
+                        <id>resources</id>
+                        <phase>generate-resources</phase>
+                        <goals>
+                            <goal>unpack</goal>
+                        </goals>
+                        <configuration>
+                            <artifactItems>
+                                <artifactItem>
+                                    <groupId>nl.knaw.dans.easy</groupId>
+                                    <artifactId>easy-licenses</artifactId>
+                                    <version>1.0.0-SNAPSHOT</version>
```

Comment:
Extract version into property `easy-licenses.version`. (Please, use this convention and *not* ~`easy.licenses.version`~.)